melodies_monet/driver.py

```diff
@@ -389,6 +432,10 @@ def mask_and_scale(self):
                             self.obj[v].data += scale
                         elif d['unit_scale_method'] == '-':
                             self.obj[v].data += -1 * scale
+                    if 'om2oc_factor' in d:
```

Comment:
@jianheACM can you provide Colin some suggestions on defining this in the docs too?

Reply:
om2oc_factor is used when comparing WRF-Chem to AQS OC data. In WRF-Chem, we have organic aerosol concentrations (soa+poa) instead of OC. So we need to convert OM to OC by this factor. If the model only gives organic aerosol concentrations, we can set this factor in yaml file. If the model gives actual OC, we do not need set this factor.